nixos/modules/services/networking/avahi-daemon.nix

```diff
@@ -39,6 +40,15 @@ let
   '';
 in
 {
+  imports = [
+    (
+      lib.mkChangedOptionModule
+        [ "services" "avahi" "interfaces" ]
+        [ "services" "avahi" "allowInterfaces" ]
+        (config: config.services.avahi.interfaces)
+    )
```

Comment:
```suggestion
    (lib.mkRenamedOptionModule [ "services" "avahi" "interfaces" ] [ "services" "avahi" "allowInterfaces" ])
```
Is this the same?

Reply:
Yes, I hadn't seen that function before, thanks!